src/js/player.js

```diff
@@ -1276,6 +1285,32 @@ class Player extends Component {
     this.trigger('play');
   }
 
+  /**
+   * Retrigger the `ratechange` event that was triggered by the {@link Tech}.
+   *
+   * If there were any events queued while the playback rate was zero, fire
+   * those events now.
+   *
+   * @private
+   * @method Player#handleTechRateChange_
+   * @fires Player#ratechange
+   * @listens Tech#ratechange
+   */
+  handleTechRateChange_() {
+    if (this.tech_.playbackRate() > 0 && this.previousPlaybackRate === 0) {
+      this.queuedCallbacks_.forEach((queued) => queued.callback(queued.event));
+      this.queuedCallbacks_ = [];
+    }
+    this.previousPlaybackRate = this.tech_.playbackRate();
```

Comment:
Should this be pseudo-private (i.e. `previousPlaybackRate_`)?

Reply:
totally missed that, yes, it should. Could even potentially be stored on the cache_ object as well/instead.

Reply:
Changed to `this.cache_.lastPlaybackRate`